doc/voq/voq_hld.md

```diff
@@ -477,7 +477,99 @@ The system port configuration has the parameters listed below.
 
 ## 2.7 CLI
 
-TO BE COMPLETED.
+CLI commands for VOQ chassis are for system ports and system neighbors. There are only show commands for these objects. As mentioned earlier, for now, since system ports are not dynamically configured and are required to be statically supplied before boot up there is no configuration commands for system ports. Since system neigobors also do not have any configurations there is no configuration commands for system neighbors. 
+
+The show commands are under **show chassis** commands group as shown below
+
+**Help info of show chassis command group**
+```
+admin@Linecard2:~$ show chassis -h
+Usage: show chassis [OPTIONS] COMMAND [ARGS]...
+
+  Chassis commands group
+
+Options:
+  -?, -h, --help  Show this message and exit.
+
+Commands:
+  modules           Show chassis-modules information
+  system-neighbors  Show VOQ system neighbors information
+  system-ports      Show VOQ system ports information
+admin@Linecard2:~$ 
+```
+### 2.7.1 System ports show command
+The information for the system ports are taken from the **SYSTEM_PORT_TABLE** of **APPL_DB** from a specific namespace 
```

Comment:
Is this show cmd to dump config or status of system port? If it dumps status, should we dump from chassis_app_db?

Reply:
This command is to show the config only.